.github/workflows/tableau-connector.yml

```diff
@@ -18,6 +27,72 @@ jobs:
         run: ./tableau-connector/build.sh
       - name: View assembled file
         run: ls -l tableau-connector/target
+      - name: Add Version to Taco file
+        run: |
+          file="./gradle.properties"
+          MAJOR_VERSION=grep "MAJOR_VERSION" ${file} | cut -d'=' -f2
+          MINOR_VERSION=grep "MINOR_VERSION" ${file} | cut -d'=' -f2
+          PATCH_VERSION=grep "PATCH_VERSION" ${file} | cut -d'=' -f2
+          mv  tableau-connector/target/documentdbjdbc.taco tableau-connector/target/documentdbjdbc-${MAJOR_VERSION}.${MINOR_VERSION}.${PATCH_VERSION}.taco
+
+      - name: Sign Executables
+        id: sign-executables-32
+        shell: bash
+        run: |
+          
+          echo "Sign is set to ${SIGNING_ENABLED}"
+          if [[ $SIGNING_ENABLED = "true" ]]
+          then
+            echo "Signing is enabled. Will attempt to sign"
+            pwd
+            ls -l tableau-connector/target
+          
+            echo "Installing jq"
+            apt-get install jq
+            name=$(ls tableau-connector/target | grep documentdbjdbc.*.taco)
+            version=$(awk -F- '{print $3}' <<<"$name")
```

Comment:
I would say that using `MAJOR_VERSION`, `MINOR_VERSION`, and `PATCH_VERSION` directly would be safer to get the version string. You could export it from the previous step, or just make the previous step code part of the this step.

Reply:
I didn't find a way to export an environment variable from one step to another step. Do you have any examples?

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable

```
echo "version=$MAJOR_VERSION.$MINOR_VERSION.$PATCH_VERSION" >> $GITHUB_ENV
```

```
... ${{env.version}} ...
```